### PR TITLE
Pick non-postgres-related changes from postgres branch

### PR DESCRIPTION
**Pull request by hungphan227**

### CODE REVIEW

**Review by Arsnael — COMMENTED** [review]
No proper git history / descriptions? Looks like there is a few different changes mingled together here

**Comment by hungphan227** [comment]
> No proper git history / descriptions? Looks like there is a few different changes mingled together here

Updated

**Review by Arsnael — APPROVED** [review]

File: tmail-backend/tmail-third-party/openpaas/pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <groupId>com.linagora.tmail</groupId>
         <artifactId>tmail-third-party</artifactId>
         <version>1.0.0-SNAPSHOT</version>
-        <relativePath>../../pom.xml</relativePath>
+        <relativePath>../pom.xml</relativePath>
```
Comment (vttranlina) [inline tmail-backend/tmail-third-party/openpaas/pom.xml:10]: like  👍  recently I got related exception

File: tmail-backend/rate-limiter/rate-limiter-api/src/test/scala/com/linagora/tmail/rate/limiter/api/RateLimitingPlanUserRepositoryContract.scala [hunk 2]
```diff
@@ -12,8 +12,8 @@ import reactor.core.scala.publisher.{SFlux, SMono}
 import scala.jdk.CollectionConverters._
 
 object RateLimitingPlanUserRepositoryContract {
-  val BOB: Username = Username.of("BOB")
-  val ALICE: Username = Username.of("ALICE")
+  val BOB: Username = Username.of("BOB@domain1.tld")
```
Comment (vttranlina) [inline tmail-backend/rate-limiter/rate-limiter-api/src/test/scala/com/linagora/tmail/rate/limiter/api/RateLimitingPlanUserRepositoryContract.scala:15]: I feel it is incorrect changes,
 are u sure?

File: tmail-backend/integration-tests/jmap/distributed-jmap-integration-tests/src/test/java/com/linagora/tmail/james/DistributedLinagoraSecondaryBlobStoreTest.java [hunk 3]
```diff
@@ -249,6 +250,8 @@ void sendEmailShouldResultingInSavingDataToBothObjectStorages(GuiceJamesServer s
             });
     }
 
+    //TODO Fix flaky test
```
Comment (vttranlina) [inline tmail-backend/integration-tests/jmap/distributed-jmap-integration-tests/src/test/java/com/linagora/tmail/james/DistributedLinagoraSecondaryBlobStoreTest.java:253]: We can note: `https://github.com/linagora/tmail-backend/issues/1299` 
for avoid forgot it

File: tmail-backend/rate-limiter/rate-limiter-api/src/test/scala/com/linagora/tmail/rate/limiter/api/RateLimitingPlanUserRepositoryContract.scala (hunk 2 above)
Comment (hungphan227) [reply tmail-backend/rate-limiter/rate-limiter-api/src/test/scala/com/linagora/tmail/rate/limiter/api/RateLimitingPlanUserRepositoryContract.scala:15]: as I remember, postgres rls test fail when these username don't have domain

**Review by vttranlina — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Pull request merged**